tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "allowSyntheticDefaultImports": true,
     "baseUrl": "./src",
     "checkJs": false,
-    "lib": ["es2017"],
+    "lib": ["es2019"],
```

Comment:
Not sure why my change led to [these type failures](https://app.circleci.com/pipelines/github/artsy/metaphysics/10519/workflows/2c97577b-e08d-434d-b17d-b1573efba310/jobs/21520), but bumping this target as instructed by the warnings worked.